.github/feature_request.md

```diff
@@ -1,16 +0,0 @@
-**Is your feature request related to a problem? Please describe.**
```

Comment:
Should also stay 👍

Reply:
Again, has been moved to ISSUE_TEMPLATE. Should this is be in `.github` folder?

Reply:
Ah didn’t see it there - then it’s good and the other one too